src/development/LocalTest/Services/LocalApp/Implementation/LocalAppHttp.cs

```diff
@@ -79,5 +87,49 @@ public LocalAppHttp(IHttpClientFactory httpClientFactory, IOptions<LocalPlatform
 
             return ret;
         }
+
+        public async Task<Instance?> Instanciate(string appId, Instance instance, string xmlPrefill, string xmlDataId)
+        {
+            var requestUri = $"{appId}/instances";
+            var serializedInstance = JsonSerializer.Serialize(instance, new JsonSerializerOptions { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault });
+
+            var content = new MultipartFormDataContent();
+            content.Add(new StringContent(serializedInstance, System.Text.Encoding.UTF8, "application/json"), "instance");
+            if (!string.IsNullOrWhiteSpace(xmlPrefill) && xmlDataId is not null)
+            {
+                content.Add(new StringContent(xmlPrefill, System.Text.Encoding.UTF8, "application/xml"), xmlDataId);
+            }
+
+            using var message = new HttpRequestMessage(HttpMethod.Post, requestUri);
+            message.Content = content;
+            // TODO: Figure out how to get orgnumber for app owner from appId
+            message.Headers.Authorization = new ("Bearer", GetOrgToken(appId.Split("/")[0], "840747972"));
```

Comment:
TODO: Obvuisly hard coding the org number of Finanstilsynet, is probably not a good strategy here. Any suggestions how to transform `"krt"` => `"840747972"` in a way that works for other orgs as well?

Reply:
I'm not entirely sure, but I believe the organisation number is unnecessary.

Reply:
I'm pretty sure it is required. I obviously need to instantiate as the correct org (not `Sophie Salt`), because it reveals typical issues in authorization rules and the use of user tokens when instantiating.

Reply:
Yeah, you need an org token. Specifically the Org claim. Just not so sure you need the OrgNumber claim.

Reply:
Ok, Finally I remember (it's been a few months). It was required in order for the instance picker that you activate in https://docs.altinn.studio/app/development/configuration/startup/ to not crash. The instance will have a null someplace unexpected if you don't set the orgnumber claim.

Reply:
We need some sort of api to lookup the org number in [GetTestOrgClaim](https://github.com/ivarne/altinn-studio/blob/5e04aaedc26e352720cb64e7ee0befc3bb5a2a90/src/development/LocalTest/Controllers/HomeController.cs#L221) so that the orgnumber claim is always set.

Reply:
Ooo, I found https://altinncdn.no/orgs/altinn-orgs.json. Is that something I could use?

Reply:
I think this should be fixed in [43fa1bb](https://github.com/Altinn/altinn-studio/pull/8563/commits/43fa1bb5c69fb9e70396bd7b9843c0678e5aa6be)

I consolidated the token generation into `AuthenticationSerivce` and standardized on 960 minutes token life (16 hours) instead of `1337` minutes for the test token apis previously.